src/frameworks.js

```diff
@@ -19,18 +19,21 @@
 'use strict';
 
 var useRequireEnsure = false;
-if (typeof module !== 'undefined' && module.require) {
-  // node.js - disable worker and set require.ensure.
-  PDFJS.disableWorker = true;
-  if (typeof require.ensure === 'undefined') {
-    require.ensure = require('node-ensure');
+if (typeof module !== 'undefined' && module.exports !== 'undefined') {
+  if (module.require) {
+    // node.js - disable worker and set require.ensure.
+    PDFJS.disableWorker = true;
+    if (typeof require.ensure === 'undefined') {
+      require.ensure = require('node-ensure');
+    }
+    useRequireEnsure = true;
+  } else if (typeof __webpack_require__ !== 'undefined') {
+    // Webpack - get/bundle pdf.worker.js as additional file.
+    PDFJS.workerSrc = require('entry?name=[hash]-worker.js!./pdf.worker.js');
+    useRequireEnsure = true;
+  } else {
+    PDFJS.workerSrc = workerify './pdf.worker.js'; // jshint ignore:line
```

Comment:
I misunderstood you question. It will throw a syntax error when used with webpack or another bundler. So yeah, we can't use it like this or it will break for non-browserify users. I need to find a better solution